index.js

```diff
@@ -142,7 +215,23 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+  let biggest = movies[0];
```

Comment:
We should call this `biggestBoxOffice`, and why not set it equal to movies[0].boxOffice?